src/utils/TimeSlots.js

```diff
@@ -19,7 +19,7 @@ export function getSlotMetrics({ min: start, max: end, step, timeslots }) {
     1 + dates.diff(start, end, 'minutes') + getDstOffset(start, end)
```

Comment:
Wouldn't it be better to handle the +1 here? Won't removing it cause any problems?

Reply:
Although I'm not completely certain, I think the `totalMin` value is used for positioning items in the calendar as well while the `numGroups` is only used for figuring out the amount of slot groups. So the impact of changing the `numGroups` seems less risky to me.
Plus, my PR is inspired by the changes suggested in https://github.com/jquense/react-big-calendar/issues/1685 and an internal patch a colleague applied for fixing this behavior. And they both have the `- 1 ` expression in the `numGroups` value and its effect is working for us quite nicely.